scripts/get_github_labels.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/python
+'''
+Fetches the labels of an Issue or Pull-Request from GitHub.
+
+Parameters:
+ --id=<id>: the number of the the GitHub Issue or Pull-Request
+ --has-label=<label>: label to check (exit 0 if set, 1 unset), without
+                      --has-label, all labels of the Issue or Pull-Request
+                      get printed.
+'''
+
+import argparse
+import sys
+import requests
+
+LABEL_URL_FMT = 'https://api.github.com/repos/ceph/ceph-csi/issues/%s/labels'
+'''
+Formatted URL for fetching the labels, replace '%s' with the number of  the
+Issue or Pull-Request.
+'''
+
+def get_json_labels(gh_id):
```

Comment:
Please add an extra line the function definition. (2 blank lines recommended as per PEP8 standards)

Reply:
Oh, really? `pylint` did not complain about that :shrug: